README.md

```diff
@@ -315,16 +315,27 @@ If anything is not working, check the code in this repository (particularly the
 }
 ```
 
-## Committing our changes
+## Afraid of Commitment?
 
 If you're using Git and GitHub, then there's one last step. Let's "commit" our changes and push them up to GitHub.
 
-First, you'll "stage" all the changes. That tells git to include them in the commit. You have to stage your changes before you can commit them:
+First let's see what files have been created or altered:
```

Comment:
Not a bad idea. I will redo the image in the same style as the others for consistency.